### PR TITLE
Update Read.md

### PR DESCRIPTION
**Pull request by DerekRoy**
This update focused on the read me file. Changes were made to improve flow, fix grammar, and promote consistency throughout the document. The goal was to create a better user experience when reading through the file.

### CODE REVIEW

**Comment by DerekRoy** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -94,35 +93,35 @@ Run your application:
...
-#### Q: What was the motivation behind this project when we already have Electron and NW.js? How this differs from these platforms, how it helps to achieve something that's not possible/harder with these two?
+#### Q: What was the motivation behind this project when we already have Electron and NW.js? How does this project differ from these platforms, how does it achieve something that is not possible/harder with Electron or NW.js?
 
-- One of the motivations is to demonstrate how browser installed locally can be used with Node out of the box.
-- Unlike with Electron, Node v8 and Chrome v8 engines are decoupled in Carlo, providing a maintainable model with the ability of the independent updates of the underlying components. Carlo is less about branding and is more about productivity + giving the control over bundling to the user.
+- One of the motivations of this project is to demonstrate how browsers that are installed locally can be used with Node out of the box.
+- Engines are decoupled in Carlo, providing a maintainable model with the ability to independently update underlying components. Carlo gives the user control over bundling and is more about productivity than branding.
```
Comment (pavelfeldman) [inline README.md]: It is essential that we mention which engines (Chrome v8 + Node v8) this refers to. Electron provides additional information to those familiar with it as well.

Comment (DerekRoy) [reply README.md]: I added those back in as "Engines like Chrome v8 and Node v8 are decoupled in Carlo".  I hope this is acceptable.

File: README.md [hunk 2]
```diff
@@ -94,35 +93,35 @@ Run your application:
...
+#### Q: What was the motivation behind this project when we already have Electron and NW.js? How does this project differ from these platforms, how does it achieve something that is not possible/harder with Electron or NW.js?
 
-- One of the motivations is to demonstrate how browser installed locally can be used with Node out of the box.
-- Unlike with Electron, Node v8 and Chrome v8 engines are decoupled in Carlo, providing a maintainable model with the ability of the independent updates of the underlying components. Carlo is less about branding and is more about productivity + giving the control over bundling to the user.
+- One of the motivations of this project is to demonstrate how browsers that are installed locally can be used with Node out of the box.
+- Engines like Node v8 and Chrome v8 are decoupled in Carlo, providing a maintainable model with the ability to independently update underlying components. Carlo gives the user control over bundling and is more about productivity than branding.
```
Comment (pavelfeldman) [inline README.md]:
```suggestion
- Node v8 and Chrome v8 engines are decoupled in Carlo, providing a maintainable model with the ability to independently update underlying components. Carlo gives the user control over bundling and is more about productivity than branding.
```

Comment (DerekRoy) [reply README.md]: Added changes.

File: README.md [hunk 3]
```diff
@@ -15,16 +14,16 @@ Carlo communicates with the locally-installed browser instance using the [Puppet
 <!-- [START usecases] -->
 ###### What can I do?
 
-With Carlo, you can create hybrid applications that use Web stack for rendering and Node for capabilities:
-- For Node applications, you can visualize dynamic state of your Node app using web rendering stack
-- For Web applications, you can expose additional system capabilities accessible from Node
-- You can bundle your application into a single executable using [pkg](https://github.com/zeit/pkg).
+Carlo creates hybrid applications that use Web stack for rendering and Node for capabilities:
```
Comment (pavelfeldman) [inline README.md]: Carlo does not create applications, it is a library. User is creating applications with Carlo.

File: README.md [hunk 4]
```diff
@@ -15,16 +14,16 @@ Carlo communicates with the locally-installed browser instance using the [Puppet
...
+- For Node applications, the web rendering stack visualizes the dynamic state of the app.
```
Comment (pavelfeldman) [inline README.md]: Similarly, web rendering stack does nothing, only user of the Carlo framework can do this.

File: README.md [hunk 5]
```diff
@@ -94,35 +93,35 @@ Run your application:
...
+#### Q: What was the motivation behind this project when we already have Electron and NW.js? How does this project differ from these platforms, how does it achieve something that is not possible/harder with Electron or NW.js?
 
-- One of the motivations is to demonstrate how browser installed locally can be used with Node out of the box.
-- Unlike with Electron, Node v8 and Chrome v8 engines are decoupled in Carlo, providing a maintainable model with the ability of the independent updates of the underlying components. Carlo is less about branding and is more about productivity + giving the control over bundling to the user.
+- One of the motivations of this project is to demonstrate how browsers that are installed locally can be used with Node out of the box.
+- Node v8 and Chrome v8 engines are decoupled in Carlo, providing a maintainable model with the ability to independently update underlying components. Carlo gives the user control over bundling and is more about productivity than branding.
```
Comment (pavelfeldman) [inline README.md]: 'more about productivity than branding' was referring to Electron, now that Electron is out of the context, it no longer makes sense.

File: README.md (hunk 3 above)
Comment (DerekRoy) [reply README.md]: Switched to "With Carlo users can create"

File: README.md (hunk 4 above)
Comment (DerekRoy) [reply README.md]: Switched to "the web rendering stack lets users visualize"

**Pull request merged**